code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1188,14 +1188,14 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 
 		var/atk_verb = user.dna.species.attack_verb
 		if(!(target.mobility_flags & MOBILITY_STAND))
-			atk_verb = "kick"
+			atk_verb = ATTACK_EFFECT_KICK
 
-		switch(atk_verb)
-			if("kick")
+		switch(atk_verb)//this code is really stupid but some genius apparently made "claw" and "slash" two attack types but also the same one so it's needed i guess
```

Comment:
Not sure I understand the comment here but if they're the same do you think 1196 should be 

`if(ATTACK_EFFECT_SLASH || ATTACK_EFFECT_CLAW)`

??

Reply:
Doesn't really fix the issue at hand, plus out of scope

Reply:
Regardless the claw attack should probably do the claw animation instead of punch, I'd consider that an oversight/bug that can be fixed within this PR.

Reply:
It does not do a punch animation

Reply:
If `ATTACK_EFFECT_CLAW` or `"claw"` is used in `dna.species.attack_verb` it will use the punch attack animation according to this block of code.

Reply:
Ah, I see what you mean.